src/app/technical-record/applicant/edit/applicant-edit.component.ts

```diff
@@ -28,20 +32,14 @@ export class ApplicantEditComponent implements OnInit {
   ngOnInit() {
     this.techRecordFg = this.parent.form.get('techRecord') as FormGroup;
 
-    const applicantdDetails: Applicant = !!this.applicant ? this.applicant : ({} as Applicant);
+    const applicantEntity: Applicant = !!this.applicant ? this.applicant : ({} as Applicant);
+
+    const applicantOnlyFields = { name: this.fb.control(applicantEntity.name) };
+    const commonAddressFields = this.commonAddressForm.createControls(applicantEntity);
 
     this.techRecordFg.addControl(
       'applicantDetails',
-      this.fb.group({
-        name: this.fb.control(applicantdDetails.name),
-        address1: this.fb.control(applicantdDetails.address1),
-        address2: this.fb.control(applicantdDetails.address2),
-        postTown: this.fb.control(applicantdDetails.postTown),
-        address3: this.fb.control(applicantdDetails.address3),
-        postCode: this.fb.control(applicantdDetails.postCode),
-        telephoneNumber: this.fb.control(applicantdDetails.telephoneNumber),
-        emailAddress: this.fb.control(applicantdDetails.emailAddress)
-      })
+      this.fb.group({ ...applicantOnlyFields, ...commonAddressFields })
```

Comment:
use `addressFields` instead of `common`.... Can this work 
` this.techRecordFg.addControl(
    'applicantDetails',
     this.fb.group({
      name: this.fb.control(applicantdDetails.name),
      ...commonAddressFields
})`